assets/src/edit-story/components/library/panes/text/textTab.js

```diff
@@ -82,8 +81,7 @@ function TextTab(props) {
   const handleAddText = (evt) => {
     evt.stopPropagation();
     insertElement('text', {
-      content: __('Fill in some text', 'web-stories'),
-      fontSize: DEFAULT_FONT_SIZE,
+      ...DEFAULT_PRESET,
       width: DEFAULT_ELEMENT_WIDTH,
```

Comment:
Does this constant's value need to be updated? According to the issue description / Figma text elements should be 360 wide.

Reply:
As per discussion on slack we should split this part into a new issue.